test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3452,6 +3454,7 @@ def forward(self, input):
         model = StandardDeviationUnbiased()
         self.run_test(model, x)
 
+    @skipIfUnsupportedMaxOpsetVersion(17)
```

Comment:
Will be merged. These are the "higher opset skips" until proper impls of opset 18+ (Reduce, etc) are added.

I'd recommend creating a tracking issue and add them as tasks.

Reply:
https://github.com/pytorch/pytorch/issues/119534

Reply:
Can we capture the error and return proper message for users while #119534 is not fixed? Something like "The model contains Reduce operators, which are not supported by ONNX Opset versions >= 18. Consider using Opset 17"

Reply:
do not know how to do this. right now if someone export a model with to version 19 or 20 and if the model contains ops that are new or updated after version 17, it will just fail. I think this is the situation not specific to symbolic_opset19 and 20. How so we handle this previously? if not all ops are added at once.